rust/agama-dbus-server/src/questions.rs

```diff
@@ -227,10 +239,35 @@ impl Questions {
         Ok(())
     }
 
-    /// sets questions to be answered by default answer instead of asking user
-    async fn use_default_answer(&mut self) -> Result<(), Error> {
-        log::info!("Answer questions with default option");
-        self.answer_strategies.push(Box::new(DefaultAnswers {}));
+    /// property that defines if questions is interactive or automatically answered with
+    /// default answer
+    #[dbus_interface(property)]
+    fn interactive(&self) -> bool {
+        let last = self.answer_strategies.last();
+        if let Some(real_strategy) = last {
+            real_strategy.id() == DefaultAnswers::id()
+        } else {
+            false
```

Comment:
As I understand it, this should be `true`

Reply:
yes, right. My fault.